templates/app/models/index.js

```diff
@@ -18,6 +18,18 @@ let databaseOptions = {
   dialectOptions: {}
 };
 
+<%if(config.dbDialect === 'mysql') {
+%>databaseOptions.dialectOptions.typeCast = (field, useDefaultTypeCasting) => {
+  if ((field.type === "BIT") && (field.length === 1)) {
+    const bytes = field.buffer();
+    return bytes[0] === 1;
+  }
+
+  return useDefaultTypeCasting();
+};
+<%}
+%>
+
```

Comment:
I'm not sure, but it seems Lumber now uses Handlebars: https://github.com/ForestAdmin/lumber/pull/333.

Reply:
Indeed you are right. It would be nice to rewrite the file and use handlebars instead. However I would rather prefer to limit the scope of this PR, and focus on the fix instead of rewriting the whole file.